spec/use_cases/reservations/cancel_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Reservations::UseCases::Cancel do
+  describe ".call" do
+    let(:seats) { [10, 11, 12, 13] }
+    let(:user) { create(:user) }
+    let(:movie) { create(:movie) }
+    let(:hall) { create(:hall) }
+    let(:screening) { create(:screening, hall_id: hall.id, movie_id: movie.id, start_time: Date.current + 2.days) }
```

Comment:
do not assign id's, assign objects, it's much cleaner
```suggestion
    let(:screening) { create(:screening, hall:, movie:, start_time: Date.current + 2.days) }
```

Reply:
same in other let's and other spec files